x-team/auto-ui
# Add on click callbacks for applicant badge actions

**Release Type:** *New feature* 


Fixes x-team/auto#1294

## Description

- Adds on click callbacks for applicant badge actions.

## Checklist



## Steps to Test or Reproduce

- Navigate to the `ApplicantBadge` component section.
  - Make sure the component passes regression.

## Related PRs

branch | PR
------ | ------
`1294-exclude-registered-from-list--release-aug-6-12` | [Auto](https://github.com/x-team/auto/pull/1447)
`1294-exclude-registered-from-list--release-aug-6-12` | [Auto API](https://github.com/x-team/auto-api/pull/466)
